pkgs/applications/misc/sakura/default.nix

```diff
@@ -32,5 +44,8 @@ stdenv.mkDerivation rec {
       terminals in one window and adds a contextual menu with some basic
       options. No more no less.
     '';
-  };
+    license     = licenses.gpl2;
```

Comment:
`gpl3` is unclear, use `gpl3Plus` or `gpl3Only`. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.

Reply:
It is GPL2Only